app/helpers/questions_helper.rb

```diff
@@ -1,2 +1,10 @@
 module QuestionsHelper
+  def question_form_header(question, form_type)
+    case form_type
+    when 'create'
+      return 'Add new question'
```

Comment:
```suggestion
      'Add new question'
```
в последней исполняемой строке метода явный ретерн не нужен